src/main/kotlin/no/nav/klage/search/api/controller/KlagebehandlingListController.kt

```diff
@@ -92,37 +91,9 @@ class KlagebehandlingListController(
         }
     }
 
-    private fun canSeeTildelteOppgaver() = innloggetSaksbehandlerRepository.erLeder() ||
-            innloggetSaksbehandlerRepository.erFagansvarlig() || true
+    private fun canSeeTildelteOppgaver() = innloggetSaksbehandlerRepository.isLeder() ||
+            innloggetSaksbehandlerRepository.isFagansvarlig() || true
 
-    //FIXME remove when not in use anymore
```

Comment:
Kult, jeg visste ikke at dette var borte allerede. Flott å få slettet da.

Reply:
Jeg regner med det iaf.. Det gamle søket fungerte ikke som det skulle, så har jo skrevet om til to ulike søk. Et for navn og et for fnr.